runtime/misc.c

```diff
@@ -28,8 +28,10 @@ __declspec(noreturn) void __cdecl abort(void);
 #include <stdio.h>
 #include <string.h>
 #include <stdarg.h>
+#if defined(DEBUG) || defined(NATIVE_CODE)
```

Comment:
@ctk21 asked about this: I just guarded the includes introduced in https://github.com/ocaml-multicore/ocaml-multicore/commit/a8f1f6cd669#diff-38bae17d49966000ccbcbf7e7660416400d6503c999207067902ea9f55cdd0c5